pkNX.WinForms/Subforms/PokeDataUI8a.cs

```diff
@@ -158,7 +158,7 @@ private void InitLearn()
             FlatStyle = FlatStyle.Flat,
         };
         dgvMove.DefaultCellStyle.Alignment = DataGridViewContentAlignment.MiddleLeft;
-        dgvMove.Items.AddRange(movelist.ToImmutableSortedSet().ToArray()); // add only the Names
```

Comment:
I think I did the sorted set in between for a reason, to lazily sort and remove duplicates